example/components/linux.py

```diff
@@ -0,0 +1,135 @@
+"""Giraffe component is the API to Giraffe component."""
+from typing import Any, Optional, TextIO, List, NamedTuple
+
+import asyncio
+import click
+import colored
+import contextlib
+import time
+import os
+
+from Octavius.lego.components import RPyCComponent
+
+
+class LinuxRPyCComponent(RPyCComponent):
+    """An extended interface for RPyC component from linux type."""
+
+    def __init__(self, *args, **kwargs):
+        """Initializes a linux RPyC component."""
```

Comment:
No need to add a summary for `__init__`, everyone knows what it does.

Reply:
OK